src/unit_tests/dataset_storage_test.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+
+def test_get_dataset_validate_keys(dataset_storage):
+    """
+    This method checks if the 'validate_keys' method ensures that each mandatory key is present in the dataset JSON.
+    """
+    with open("../test_data/dataset.json") as f:
```

Comment:
Minor theres now a config variable that you could use here instead

Reply:
@AND-JAshcroft Thanks Joe, just corrected this.